lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_SOUND_LOUD c=7
+#: messages.c:144
+msgid "Loud"
+msgstr "H\xe1gt"
+
+# MSG_SLIGHT_SKEW c=14
+#: ultralcd.cpp:2887
+msgid "Slight skew"
+msgstr "Lag skevhet"
+
+# MSG_SOUND c=7
+#: messages.c:143
+msgid "Sound"
+msgstr "Ljud"
+
+# MSG_RUNOUTS c=7
+#: ultralcd.cpp:1593
+msgid "Runouts"
+msgstr "Avbrott"
+
+# MSG_Z-LEVELING_ENFORCED c=20 r=4
+#: Marlin_main.cpp:3303
+msgid "Some problem encountered, Z-leveling enforced ..."
+msgstr "Ett problem har uppstatt, Z-nivellering patvingad..."
+
+# MSG_SOUND_ONCE c=7
+#: messages.c:145
+msgid "Once"
+msgstr "En gang"
+
+# MSG_SPEED c=15
+#: ultralcd.cpp:6882
+msgid "Speed"
+msgstr "Fart"
+
+# MSG_SELFTEST_FAN_YES c=19
+#: messages.c:93
+msgid "Spinning"
+msgstr "Rotation"
+
+# MSG_TEMP_CAL_WARNING c=20 r=4
+#: Marlin_main.cpp:5351
+msgid "Stable ambient temperature 21-26C is needed a rigid stand is required."
+msgstr "Stabil omgivningstemperatur 21-26C kr\xe1vs samt ett styvt stativ."
+
+# MSG_STATISTICS c=18
+#: ultralcd.cpp:6081
+msgid "Statistics"
+msgstr "Statistik"
+
+# MSG_STOP_PRINT c=18
+#: messages.c:110
+msgid "Stop print"
+msgstr "Stoppa utskriften"
+
+# MSG_STOPPED c=20
+#: messages.c:111
+msgid "STOPPED."
+msgstr "STOPPAD."
+
+# MSG_SUPPORT c=18
+#: ultralcd.cpp:6756
+msgid "Support"
+msgstr "Support"
+
+# MSG_SELFTEST_SWAPPED c=16
+#: ultralcd.cpp:8180
+msgid "Swapped"
+msgstr "Bytt"
+
+# MSG_SELECT_FILAMENT c=20
+#: ultralcd.cpp:4706
+msgid "Select filament:"
+msgstr "V\xe1lj filament:"
+
+# MSG_TEMP_CALIBRATION c=14
+#: messages.c:112
+msgid "Temp. cal."
+msgstr "Temp. kal."
+
+# MSG_SELECT_TEMP_MATCHES_MATERIAL c=20 r=4
+#: ultralcd.cpp:4847
+msgid "Select temperature which matches your material."
+msgstr "V\xe1lj temperatur som passar ditt material."
+
+# MSG_CALIBRATION_PINDA_MENU c=17
+#: ultralcd.cpp:5812
+msgid "Temp. calibration"
+msgstr "Temp. kalibrering"
+
+# MSG_TEMP_CAL_FAILED c=20 r=8
+#: ultralcd.cpp:3771
+msgid "Temperature calibration failed"
+msgstr "Temperaturkalibrering misslyckades"
+
+# MSG_TEMP_CALIBRATION_DONE c=20 r=12
+#: messages.c:113
+msgid "Temperature calibration is finished and active. Temp. calibration can be disabled in menu Settings->Temp. cal."
+msgstr "Temperaturkalibreringen \xe1r klar och aktiv. Temp. kalibrering kan inaktiveras i menyn Inst\xe1llningar->Temp. kal."
+
+# MSG_FS_VERIFIED c=20 r=3
+#: ultralcd.cpp:7411
+msgid "Sensor verified, remove the filament now."
+msgstr "Sensor verifierad, ta bort filamentet nu."
+
+# MSG_TEMPERATURE c=18
+#: ultralcd.cpp:5673
+msgid "Temperature"
+msgstr "Temperatur"
+
+# MSG_MENU_TEMPERATURES c=15
+#: ultralcd.cpp:2047
+msgid "Temperatures"
+msgstr "Temperaturer"
+
+# MSG_FOLLOW_Z_CALIBRATION_FLOW c=20 r=9
+#: messages.c:47
+msgid "There is still a need to make Z calibration. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Det finns fortfarande ett behov av att g\xefra Z-kalibrering. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.